src/screens/Learn/End/End.jsx

```diff
@@ -0,0 +1,97 @@
+import React, { useMemo, useState, useEffect } from "react";
+import { useTranslation } from "react-i18next";
+import { useSelector, useDispatch } from "react-redux";
+import { useHistory } from "react-router-dom";
+
+import Button from "../../../Components/Button/Button.jsx";
+import Table from "../../../Components/Table/Table.jsx";
+
+import { clearQuery } from "../../../redux/Actions/query.js";
+
+import "./End.scss";
+
+const End = () => {
+  const correctVocabs = useSelector((state) => state.query.correct);
+  const wrongVocabs = useSelector((state) => state.query.wrong);
+  const [percentage] = useState(
```

Comment:
why is this a state?

Reply:
is this resolved? I could change it to a normal const variable